proto/src/determined/api/v1/api.proto

```diff
@@ -1381,6 +1381,7 @@ service Determined {
   rpc MetricNames(MetricNamesRequest) returns (stream MetricNamesResponse) {
     option (google.api.http) = {
       get: "/api/v1/experiments/{experiment_id}/metrics-stream/metric-names"
+      // get: "/api/v1/experiments/{experiment_id}/metric-names"
```

Comment:
can uncomment this to make sure errors are properly handled in this kind of situation as well.